### PR TITLE
Update to net6.0

### PR DESCRIPTION
**Pull request by abheekda1**
I used the upgrade assistant along with some manual changes and it compiles with no errors and no warnings and seems to be functional. More testing may need to be done but for now it seems to be working.

### CODE REVIEW

**Comment by abheekda1** [comment]
Note: it isn't launching after finding the EXEs for the tools, it just hangs.

**Comment by abheekda1** [comment]
Seems the issue comes during the loading of the main form -- the conflict tree specifically. See if I can figure some stuff out regarding that.

File: WitcherScriptMerger.backup/Controls/SMTree.cs [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+            // contextCollapseAll
+            _contextCollapseAll.Name = "contextCollapseAll";
+            _contextCollapseAll.Size = new Size(225, 22);
+            _contextCollapseAll.Text = "Collapse All";
+            _contextCollapseAll.Click += ContextCollapseAll_Click;
+        }
+
+        protected void BuildContextMenu()
+        {
+            _contextMenu.Items.Clear();
```
Comment (abheekda1) [inline WitcherScriptMerger.backup/Controls/SMTree.cs]: This seems to be the offending line.

Comment (abheekda1) [reply WitcherScriptMerger.backup/Controls/SMTree.cs]: Ok yeah it's fixed now.

**Comment by abheekda1** [comment]
@IDCs I think this should be fully functional now and I also think that it would be worth it to update to net6.0.

**Comment by IDCs** [comment]
Sorry it took so long to review your changes - looks good thank you for your contribution!

Going to release this once I make the necessary changes to the Vortex extension so that it can pull it and configure it automatically.

**Comment by abheekda1** [comment]
Sounds great! I'd like to get the script merger as up-to-date as possible so I might be making some more PRs in the future. Thanks for taking a look and merging it!

**Pull request merged**